### ISSUE TITLE
User should be able to add exact coords to measurement, instead of using map

### ISSUE THREAD

**Issue by stringfellow** [openwater/h2o-really#22]

**Comment by stringfellow** [comment]
see add-obs-basics branch

**Comment by jimr** [comment]
What about using HTML5 geolocation, at least for default position?

**Comment by stringfellow** [comment]
Yes - I have already added this, but it's not really working so well with setting the map location (I think because of the projection being 900913)… But, even giving the map a default, the user should be able to enter lat/lng (probably in various formats). It's commented out in the js block of observations.html in observations app ATM.

**Comment by jimr** [comment]
Aye I hacked about a bit with hooking up the Google place name search but the projection was off and I didn't try too hard to fix it. If it does work though I think you can just type coordinates into the search box.
